config/config_test.go

```diff
@@ -370,16 +371,16 @@ func TestShouldInclude_IncludeWhenMatches(t *testing.T) {
 func TestShouldInclude_WhenMatchesIncludeAndExclude(t *testing.T) {
 	include, err := regexp.Compile(`test.*`)
 	require.NoError(t, err)
-	includeREs := []Expression { { RE: *include } }
+	includeREs := []Expression{{RE: *include}}
 
 	exclude, err := regexp.Compile(`.*openvpn.*`)
 	require.NoError(t, err)
-	excludeREs := []Expression { { RE: *exclude } }
+	excludeREs := []Expression{{RE: *exclude}}
```

Comment:
NIT: just noting if this was intentional - I don't think it's a massive problem, but some different formatting.